app/views/about/show.html.haml

```diff
@@ -7,9 +7,10 @@
 
 .landing
   .landing__brand
+    = image_tag '/qh-logo.png', class: 'qh-landing-logo'
```

Comment:
Please add the styling for this new element also to the `queerhaus-bright` theme, otherwise the layout will break since that is the current default theme.